intro_to_R_tidyverse/00-rstudio_guide.md

```diff
@@ -0,0 +1,80 @@
+RStudio is an [integrated development environment](https://en.wikipedia.org/wiki/Integrated_development_environment) or IDE that makes it easier to program in R. 
+In our case, we use [Docker](https://www.docker.com/) containers with RStudio on them (specifically from the [Rocker Project](https://www.rocker-project.org/)) to ensure that all course participants have the same environment.
+A detailed discussion of Docker is outside the scope of this guide.
+**Here, our objective is to provide a brief guide to getting started with RStudio.**
+We assume you have either installed R and RStudio locally or are using an appropriate Docker container.
+
+Note there is an excellent _Take a tour of RStudio’s IDE_ video from RStudio [here](https://www.rstudio.com/products/rstudio/).
+
+## Anatomy of RStudio
+
+_These screenshots were generated using a Docker container._
+
+1. When you open an RStudio session for the first time, it will look like this.
+
+![overview](screenshots/00-rstudio.png)
+
+2. The **Console** is where you will type commands that will be carried out by `R`.
+
+![console](screenshots/01-console.png)
+
+3. You can switch to a **Terminal** where commands will be carried out by clicking the `Terminal` tab to the right of the `Console` tab. 
+This is a [command-line interface](https://en.wikipedia.org/wiki/Command-line_interface).
+Commands here will not be carried out by R.
+
+![terminal](screenshots/02-terminal.png)
+
+4. The **Environment** pane displays all of the objects and functions in our global environment.
+Here we can see that we have a `data.frame` called `iris` in our global environment.
+
+![environment](screenshots/03-environment.png)
+
+5. You can click on `iris` to see what the `data.frame` looks like.
+This will open a `View` in the Source pane.
+
+![view](screenshots/04-view.png)
+
+6. The **Plots** pane will display any plots you make during your session.
+Here we've made a scatter plot with the `iris` data.
+
+![plots](screenshots/05-plots.png)
+
+7. The **Help** pane gives you access to documentation about R packages and functions.
+Typing `?plot` into the R **Console**  opens up the information about this function in the **Help** pane.
+
+![help](screenshots/06-help.png)
+
+8. The **Packages** pane tells you what packages are installed and which version.
+Clicking on a package name will bring up its documentation in the **Help** pane.
+
+![packages](screenshots/07-packages.png)
+
+9. The **History** pane displays all the commands that have been executed in R.
+
+![history](screenshots/08-history.png)
+
+10. The **Files** pane displays files for whatever directory or folder it is currently pointed at. 
+Whatever the **Files** pane is pointed at is not automatically set as your working directory.
+Try typing `?setwd` and `?getwd` to learn more about working directories.
+
+![files](screenshots/09-files.png)
+
+11. Clicking an R Markdown file (`.Rmd`) in the **Files** pane will open it up in the **Source** pane.
```

Comment:
May want to note that other scripts like regular Rscripts will also open up in the same way.